### PR TITLE
chore: Add GetTransactions Benchmarks

### PR DESCRIPTION
**Pull request by antoinegelloz**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -30,6 +30,32 @@ jobs:
   Test_sqlite:
     uses: formancehq/gh-workflows/.github/workflows/golang-test.yml@main
 
+  bench:
```
Comment (gfyrag) [inline .github/workflows/main.yml]: Bench are very slow, I don't think we have to add them to the CI.
@flemzord ? [](edited-with-Axolo)

File: pkg/storage/sqlstorage/transactions_test.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+					},
+					acc: {
+						"USD": {
+							Input:  core.NewMonetaryInt(0),
+							Output: core.NewMonetaryInt(0),
+						},
+					},
+				},
+			}
+			err := store.Commit(context.Background(), tx)
```
Comment (gfyrag) [inline pkg/storage/sqlstorage/transactions_test.go]: Insertions is very slow when inserting transactions one by one.
Use a bench ;) [](edited-with-Axolo)

**Review by gfyrag — COMMENTED** [review]
I don't think we need to add bench tests to the CI.
It will bloat the CI on the long run.
It's already pretty slow.

File: .github/workflows/main.yml (hunk 1 above)
Comment (antoinegelloz) [reply .github/workflows/main.yml]: I added the benchmarks in a separate workflow_dispatch, to run them only manually. What do you think?

**Review by jdupas22 — APPROVED** [review]

**Review by darkmatterpool — APPROVED** [review]

**Review by flemzord — APPROVED** [review]

**Pull request merged**